config/locales/en/casework.yml

```diff
@@ -134,7 +134,7 @@ en:
     undetermined: Undetermined
     urn: Unique reference number (URN)
     has_case_concluded: Has the case concluded?
-    date_case_concluded: When did the case conclude?
+    date_case_concluded: Date when the case concluded
```

Comment:
I think this should just be 'When the case concluded' https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=8359%3A48221&mode=design&t=2WRV0zh0YlF5eUrj-1